app/src/main/java/com/sun/premierleague/data/model/NotificationMatch.kt

```diff
@@ -0,0 +1,34 @@
+package com.sun.premierleague.data.model
+
+import android.content.ContentValues
+
+data class NotificationMatch(
+    val matchId: String,
+    val matchDate: String,
+    val matchTime: String,
+    val matchHomeTeamId: String,
+    val matchHomeTeamName: String,
+    val matchAwayTeamId: String,
+    val matchAwayTeamName: String
```

Comment:
```suggestion
data class MatchNotification(
    val id: String,
    val date: String,
    val time: String,
    val homeTeamId: String,
    val homeTeamName: String,
    val awayTeamId: String,
    val awayTeamName: String
```

```kotlin
    val homeTeamId: String,
    val homeTeamName: String,
    val awayTeamId: String,
    val awayTeamName: String
```
cái này nên tạo ra class Team và có thuộc tính check xem là `home` hay `away` sẽ tốt hơn